app/models/google_event.rb

```diff
@@ -111,6 +105,20 @@ def load_emails
       ConferenceRoom.pluck(:email)
     end
 
+    GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
+    def add_events_from_room(room, service, events, config)
+      service.list_events(room.email, config) do |result, _|
+        if result
+          result.items.each do |event|
```

Comment:
I noticed you changed `result.items&.each` to `result.items.each`. Can `result.items` be nil?